### DISCUSSION TITLE
a67b023d-b40f-4619-9d3d-2490f98e153b

### DISCUSSION THREAD

**Discussion by giscus** [SSWConsulting/SSW.Rules.Content#9537] [Rules Discussions]
# a67b023d-b40f-4619-9d3d-2490f98e153b

Avoid publishing from Visual Studio and instead use a defined Build in TFS for reliable and repeatable deployments.

https://www.ssw.com.au/rules/do-you-avoid-publishing-from-visual-studio/

**Comment by wicksipedia** [comment]
Right click publish *shudder* when this happens you also have the problem that there was no log that a deployment happened! Being able to reconstruct the events that lead up to an issue is very important
